lib/views/doujin/doujin_library/doujin_cards.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rating_bar/flutter_rating_bar.dart';
+
+class DoujinCards extends StatefulWidget {
```

Comment:
The name of the class should be singular, because its just 1 DoujinCard which contains the data to one doujin.